### DISCUSSION TITLE
Betrayel

### DISCUSSION THREAD

**Discussion by Puddleglumn** [chromiecraft/chromiecraft#4520] [Suggestions]
So EQ2 had a betrayal quest.  I always felt like it would be neat for wow to have something similar.  There are plenty of allianc characters in horde lands and vice versa.  Theyre just all npcs.  Why couldnt players benefit from this?  It would be something novel to Chromie craft.

**Comment by Cooties-cc** [comment]
Seems a bit too custom perhaps. What its asking for is to allow faction changes without race swaps, from what I can see. We already have xfac guilds, BGs, grouping, and RDF, yet dunno how people would feel about fluidity of factions. With the upcoming expansion sets, this isn't really an issue, since both the main hubs for factions are sanctuary cities. It also takes a heavy toll in coding, cause now you have to change a lot of things (mainly quests) to be avail to a different race than what is possible.
